src/service/lib/get-mock-data.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const {FILE_NAME} = require(`../../const`);
+const fs = require(`fs`).promises;
+
+let data = null;
+
+const getMockData = async () => {
+  if (data !== null) {
+    return Promise.resolve(data);
+  }
+
+  try {
+    const fileContent = await fs.readFile(FILE_NAME);
+    data = JSON.parse(fileContent);
+  } catch (err) {
+    console.log(err);
+    return Promise.reject(err);
+  }
+
+  return Promise.resolve(data);
```

Comment:
```suggestion
    return data;
  }

    const fileContent = await fs.readFile(FILE_NAME);
    data = JSON.parse(fileContent);


  return data;
```
асинхронная функция всегда возвращает промисс